CHANGELOG.md

```diff
@@ -3,10 +3,19 @@
 BREAKING CHANGES
   * `client.grpc` defaults to `true` now instead of `false`. This is to make it
      harder to misconfigure Connect [[GH-282](https://github.com/hashicorp/consul-helm/pull/282)]
-     
+
      If you do not wish to enable gRPC for clients, set `client.grpc` to
      `false` in your local values file.
 
+  * `connectInject.centralConfig` defaults to `true` now instead of `false`. This is to make it
```

Comment:
This probably needs to be updated because we've released a couple of times.

Reply:
✅